lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.h

```diff
@@ -165,6 +165,11 @@ class SwiftLanguageRuntime : public LanguageRuntime {
   static FuncletComparisonResult
   AreFuncletsOfSameAsyncFunction(llvm::StringRef name1, llvm::StringRef name2);
 
+  /// See AreFuncletsOfSameAsyncFunction(StringRef, StringRef).
+  static FuncletComparisonResult
+  AreFuncletsOfSameAsyncFunction(swift::Demangle::NodePointer node1,
+                                 swift::Demangle::NodePointer node2);
+
```

Comment:
This may also fit in `TypeSystem/Swift/SwiftDemangle.h`

Reply:
I'll need to look at all of these `Are<something>Async<something>` functions and figure out where they should go